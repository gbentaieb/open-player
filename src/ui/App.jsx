import React from 'react';

const App = () => (
  <div className="playerApp">
    <div />
  </div>
);

export default App;
