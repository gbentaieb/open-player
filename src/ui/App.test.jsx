import React from 'react';
import App from './App';

describe('test App component', () => {
  test('App > Snapshot', () => {
    const wrapper = shallow(<App config={{}} onLoad={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});

