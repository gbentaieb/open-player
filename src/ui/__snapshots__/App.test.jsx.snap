// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test App component App > Snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Player
    config={Object {}}
    onLoad={[Function]}
  />
</Provider>
`;
